helm/values/csi-driver/v2.1.1/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Default values for hpe-csi-driver Helm chart
```

Comment:
We don't need a values file as well for 2.1.1. Please remove

Reply:
removed